ios/File.swift

```diff
@@ -0,0 +1,8 @@
+//
+//  File.swift
```

Comment:
Is this file needed?

Reply:
Yes, at least for now. I followed the [official steps](https://github.com/react-native-community/upgrade-support/issues/13) to upgrade the Xcode project but due to this [issue](https://github.com/react-native-community/upgrade-support/issues/25#issue-591311756), the dummy file is needed. If it works for you without the file let me know